src/parser/applyViewboxTransform.ts

```diff
@@ -148,6 +173,7 @@ export function applyViewboxTransform(element) {
     el = element.ownerDocument.createElementNS(svgNS, 'g');
     // element.firstChild != null
     while (element.firstChild) {
```

Comment:
This feels like a bug because we never change `element`. Is that done implicitly somehow? This confuses me.